src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -0,0 +1,109 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.elemnt.Wait;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.function.Function;
+
+public class AutoplayVuap {
+
+  private static final String PAUSE_CLASS_NAME = "pause-overlay";
+
+  private static final String STOP_CLASS_NAME = "close-ad";
+
+  private static final String CURRENT_TIME_CLASS_NAME = "current-time";
+
+  private static final String PAUSE_BUTTON_SELECTOR_FORMAT = "#%s ." + PAUSE_CLASS_NAME;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "#%s ." 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)